radar_patient_check/main.py

```diff
@@ -31,13 +31,19 @@ async def radar_check(nhs_number: str) -> bool:
     """
     with Session(ukrdc_engine) as session:
         if (
-            patient := session.query(PatientNumber)
+            patient_numbers := session.query(PatientNumber)
             .filter_by(patientid=nhs_number)
```

Comment:
Can you please also add a filters for `PatientNumber.numbertype = 'NI'`? I'd expect that a number with the right format for an NHS number wouldn't accidentally match a local hospital number or RADAR number, but it's worth us being extra careful I think.

Note for future reference, the distinct combinations of organisation and numberype are:

| organization | numbertype |
|--------------|------------|
| HSC          | MRN        |
| NHS          | MRN        |
| LOCALHOSP    | MRN        |
| RADAR        | MRN        |
| CHI          | MRN        |
| NHS          | NI         |
| CHI          | NI         |
| HSC          | NI         |

So if we filter to only NI numbertypes then we implicitly return only matches on NHS/CHI/HSC numbers